File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/InlineObject14.scala

package org.openapitools.client.model


case class InlineObject14 (
    /* Group ID to query. */
    _groupId: Long,
    _vehicles: List[Vehicle]
)
object InlineObject14 {
    def toStringBody(var_groupId: Object, var_vehicles: Object) =
        s"""
        | {
        | "groupId":$var_groupId,"vehicles":$var_vehicles
        | }
        """.stripMargin
}

File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/InlineObject14.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.Vehicle
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param groupId Group ID to query.
 * @param vehicles 
 */
data class InlineObject14 (

        @get:NotNull 
        @JsonProperty("groupId") val groupId: Long,

        @get:NotNull 
        @JsonProperty("vehicles") val vehicles: List<Vehicle>
) {

}

